discovery-infra/test_infra/consts/consts.py

```diff
@@ -9,6 +9,7 @@
 IMAGE_NAME = "installer-image.iso"
 STORAGE_PATH = "/var/lib/libvirt/openshift-images"
 SSH_KEY = "ssh_key/key.pub"
+VM_CPU_MODE = "host-passthrough"
```

Comment:
What do you think about using more descriptive name like `MASTER_NODE_PASS_CPU_FEATURES` or `WORKER_NODE_PASS_CPU_FEATURES`?

Reply:
yeah, OK, i agree the name is not informative enough. But MASTER_NODE_PASS_CPU_FEATURES also not clear for anyone that is out of this context.... how about MASTER_TF_CPU_MODE?

Reply:
the name you proposed sounds good

Reply:
done